.eslintrc.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+  env: {
+    browser: true,
+    jest: true,
+  },
+  extends: [
+    'plugin:react/recommended',
+    'semistandard',
+    'plugin:@typescript-eslint/eslint-recommended',
+    'plugin:@typescript-eslint/recommended',
+    'plugin:@typescript-eslint/recommended-requiring-type-checking'
+  ],
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    project: [
+      './tsconfig.json'
+    ]
+  },
+  plugins: ['react', '@typescript-eslint'],
+  settings: {
+    react: {
```

Comment:
There is also hooks - https://github.com/polkadot-js/dev/blob/master/packages/dev-react/config/eslint.js#L17